tool/tctl/common/resource_command.go

```diff
@@ -583,6 +600,11 @@ func (rc *ResourceCommand) Delete(client auth.ClientI) (err error) {
 			return trace.Wrap(err)
 		}
 		fmt.Printf("session recording configuration has been reset to defaults\n")
+	case types.KindNetworkRestrictions:
+		if err = resetNetworkRestrictions(ctx, client); err != nil {
+			return trace.Wrap(err)
+		}
+		fmt.Printf("network restrictions have been reset to defaults\n")
```

Comment:
The default is allow all right? So maybe `network restrictions have been reset to defaults (allow all)` just to the user understand the implications of what was done.

Reply:
correct, will update